clr-k8s-examples/create_stack.sh

```diff
@@ -310,6 +320,7 @@ command_handlers[storage]=storage
 command_handlers[monitoring]=monitoring
 command_handlers[metallb]=metallb
 command_handlers[npd]=npd
+command_handlers[nfd]=nfd
```

Comment:
can we also add a command_help message for nfd `command_help[nfd]="message"`, since it shows as undocumented for now `nfd:         	Not-documented` .

Reply:
We can. I have been one fence about adding everything to help and consider that more of the 'things we officially support' category. I've got to rebase and can add it. Thanks!